ui/raidboss/data/02-arr/trial/titan-nm.txt

```diff
@@ -0,0 +1,105 @@
+### Titan Story / Normal
+# -ii 5C5 3CD -p 28B:203.0  28C:603.0
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
+
+### Phase 1 100->85%
+0.0 "Start"
+0.0 "--sync--" sync /Engage!/ window 0,1
+2.0 "--sync--" sync /:Titan:368:/ window 2,0
+5.0 "Rock Buster" sync /:Titan:281:/
+7.0 "Tumult" sync /:Titan:282:/
+16.1 "Tumult" sync /:Titan:282:/
+19.2 "Rock Buster" sync /:Titan:281:/
+25.2 "Tumult" sync /:Titan:282:/ window 8,8
+
+32.3 "Rock Buster" sync /:Titan:281:/
+34.3 "Tumult" sync /:Titan:282:/
+43.4 "Tumult" sync /:Titan:282:/
+46.5 "Rock Buster" sync /:Titan:281:/
+52.5 "Tumult" sync /:Titan:282:/ window 8,8 jump 25.2
+
+59.6 "Rock Buster"
+61.6 "Tumult"
+70.7 "Tumult"
+73.8 "Rock Buster"
+79.8 "Tumult"
+
+### Phase 2 85->55%
+200.0 "--sync--" sync /:28B:Titan/ window 200,0
+203.0 "Geocrush" sync /:Titan:28B:/
+
+211.6 "Landslide" sync /:Titan:28A:/ window 211,17
+217.8 "Rock Buster" sync /:Titan:281:/
+221.8 "Tumult" sync /:Titan:282:/
+229.1 "Landslide" sync /:Titan:28A:/
+235.3 "Rock Buster" sync /:Titan:281:/
+239.3 "Tumult" sync /:Titan:282:/ window 17,16
+
+246.6 "Landslide" sync /:Titan:28A:/
+252.8 "Rock Buster" sync /:Titan:281:/
+256.8 "Tumult" sync /:Titan:282:/ window 15,17
+264.1 "Landslide" sync /:Titan:28A:/
+270.2 "Rock Buster" sync /:Titan:281:/
+274.2 "Tumult" sync /:Titan:282:/ window 17,16 jump 239.3
+
+281.5 "Landslide"
+287.6 "Rock Buster"
+291.6 "Tumult"
+298.9 "Landslide"
+305.1 "Rock Buster"
+309.1 "Tumult"

```

Comment:
Sorry, would you mind lining up this loop?

```
281.5 "Landslide" #sync /:Titan:28A:/
287.7 "Rock Buster" #sync /:Titan:281:/
291.7 "Tumult" #sync /:Titan:282:/
299.0 "Landslide" #sync /:Titan:28A:/
305.1 "Rock Buster" #sync /:Titan:281:/
309.1 "Tumult" #sync /:Titan:282:/
```

When a timeline loops, it should have the same times as where it jumps to, just adjusted forward in time.  I should really add some sort of `loop` command to do this for you, but for now there's always `util/timeline_adjust.py`.

Reply:
To expand on that a little, once you have a clean loop established, what you want to do is take the timestamp from the ability at the start of the loop and the timestamp at the next iteration's start. Subtract to find the "length" of the loop. Then run `timeline_adjust.py` with the `-a` flag set to that length value.

The reason for this is that very few encounters have perfect loops, in the sense that there's no timing variance between loop iterations. Bosses moving to do autos will introduce a little drift. This makes the timeline a bit "shaky" when jumping, since the user will see the fake loop abilities suddenly snap to the real loop abilities.